### ISSUE TITLE
Skull raid draws cards if the discarded cards go to the battlefield

### ISSUE THREAD

**Issue by mdog1** [PennyDreadfulMTG/modo-bugs#901]
**What is the expected behaviour of the card?**
 - Skull raid should not draw the player that casts it any cards if their opponent discards 2. Even if the discarded cards go to the battlefield, they still count as being discarded for effects like Liliana's caress. 

**What is the actual behaviour of the card?**
 - Skull raid draws the player that casts it 2 cards if both discarded cards go to the battlefield when discarded. I assume it would draw the caster 1 card if 1 normal card and 1 that goes to the battlefield were discarded, but have not tested this.

**Screenshot**
![image](https://user-images.githubusercontent.com/40871277/110185969-d20b4680-7de1-11eb-8ee0-b4e699b8fe6f.png)

**Please provide a list of affected cards**
Affects: Skull Raid, Loxodon Smiter, Wilt-leaf Liege

**Card Images**

**Comment by Kalioc** [comment]
Cards are only considered discarded if they enter the graveyard from hand. Cards with effects like Loxodon Smiter's are put onto the battlefield instead of being discarded.
Skull Raid will draw two cards when two Smiter-like creatures are chosen for discard because zero cards actually get discarded.

701.8a To discard a card, move it from its owner's hand to that player's graveyard.

**Comment by emerald000** [comment]
That is wrong, the card is still being discarded but is moved to another zone. You are not supposed to draw cards.

**Comment by stash86** [comment]
701.8a is clear. Discard means putting a card from hand to GY. If the card ends up in other zone than GY, it isn't considered as being discarded. The same thing as "Die" means the creature have to actually hits GY.

**Comment by emerald000** [comment]
Here's the ruling on Loxodon Smiter:

If you discard Loxodon Smiter and put it onto the battlefield, you’ve still discarded a card. Abilities that trigger when you discard a card (such as the one from Liliana’s Caress) will still trigger.

**Comment by stash86** [comment]
Re-opening the issue and gives it advantageous bug tag

**Comment by datrobino** [comment]
still bugged
